stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -379,11 +395,16 @@ inline void AsyncTask::flagAsSuspended() {
 
     if (_private().Status.compare_exchange_weak(oldStatus, newStatus,
                                                 std::memory_order_relaxed,
-                                                std::memory_order_relaxed))
+                                                std::memory_order_relaxed)) {
+      swift_task_exitThreadLocalContext((char *)&_private().ExclusivityAccessSet[0]);
       return;
+    }
   }
 }
 
+inline void AsyncTask::flagAsCompleted() {
+}
```

Comment:
This function intentionally left blank?

Reply:
Yes. It is meant to be a place where you could put logging. I was nervous about putting it in. But maybe I should?

Reply:
Might make sense to preemptively put in the logging, ifdefed out (`SWIFT_TASK_PRINTF_DEBUG` is already used elsewhere and would probably make sense here). Then it makes the purpose of this function apparent, _and_ helps out whoever needs some logging in the future.